docs/sparkr.md

```diff
@@ -210,6 +210,43 @@ head(df)
 {% endhighlight %}
 </div>
 
+### Model Formulae
```

Comment:
It should be a top-level section after `SparkR DataFrames`. We can call it `Machine Learning` and have a subsection under it called `Generalized linear models`, then move the content there. `R formulae` handling is hidden from SparkR users. They should be able to use it the same way as in R. So `Generalized linear models` is a more accurate title here. cc: @shivaram